composer.json

```diff
@@ -16,8 +16,6 @@
     "minimum-stability": "dev",
     "require-dev": {
         "symfony/symfony": "2.6",
```

Comment:
why do we require symfony/symfony here? and why 2.6? this will break lower version testing, so we actually would have to bump the minimum version in require as well if its true.

but did this not just slip in from testing the travis setup that currently requires symfony/symfony 2.6?